packages/typespec-ts/test/integration/azurecore.spec.ts

```diff
@@ -32,11 +31,23 @@ describe("Azure Core Rest Client", () => {
         },
         contentType: "application/json"
       });
+      if (isUnexpected(result)) {
+        throw Error("Unexpected status code");
+      }
       assert.strictEqual(result.status, "200");
+      if (result.status === "200") {
```

Comment:
why we need this check? is there any status code?

Reply:
I have update the test file as you said